src/controllers/user.controller.js

```diff
@@ -54,7 +54,16 @@ class UserController {
         role
       });
 
-      // TODO : SEND EMAIL TO USER
+      const { protocol } = req;
+      mailer({
+        name: username,
+        receiver: email,
+        subject: 'Welcome to ErrorSwag',
+        templateName: 'confirm_account',
+        confirm_account_url: `${protocol}://${req.get(
+          'host'
+        )}/api/v1/auth/verify/${token}`
+      });
 
```

Comment:
Good work here 👍. However, I `cloned` the `repo` and `signed up` with my `Gmail` account and received no email but I got a `response body` from `Postman` as shown below. 
![image](https://user-images.githubusercontent.com/31288326/61188058-9dc59e00-a671-11e9-8d30-9a51e5e0e303.png)

Is there something I didn't do that made me not get the email sent upon registration?

Reply:
You may need to check your spam if it's not sent to your inbox